src/encoded/static/components/home.js

```diff
@@ -710,114 +712,203 @@ ChartGallery.defaultProps = {
 };
 
 
-// Component to allow clicking boxes on classic image
-class AssayClicking extends React.Component {
-    constructor(props) {
-        super(props);
+// Defines characteristics of the svg rectangles overlaid on the classic image.
+//   queryValue: Used in queries to the server to filter assays.
+//   voice: Text for screen readers to say.
+const assayList = [
+    {
+        queryValue: '3D chromatin structure',
+        voice: '3D chromatin structure',
+    },
+    {
+        queryValue: 'DNA accessibility',
+        voice: 'Chromatin accessibility',
+    },
+    {
+        queryValue: 'DNA binding',
+        voice: 'Chromatin interactions',
+    },
+    {
+        queryValue: 'DNA methylation',
+        voice: 'Methylome',
+    },
+    {
+        queryValue: 'COMPPRED', // Special case; query hardcoded elsewhere
+        voice: 'Chromatin modification',
+    },
+    {
+        queryValue: 'Transcription',
+        voice: 'Transcriptome',
+    },
+    {
+        queryValue: 'RNA binding',
+        voice: 'RNA binding',
+    },
+];
 
-        // Required binding of `this` to component methods or else they can't see `this`.
-        this.sortByAssay = this.sortByAssay.bind(this);
-    }
+
+// Component to allow clicking boxes on classic image
+const AssayClicking = ({ assayCategory, handleAssayCategoryClick }) => {
+    const assayClickHandled = React.useRef(false);
```

Comment:
when I see "Handled" in JavaScript, I think of an event rather than a variable. If other see it that way too, they may be confused on seeing it is a variable.